usecases/api/cem_ohpcf.go

```diff
@@ -0,0 +1,14 @@
+package api
+
+import (
+	"github.com/enbility/eebus-go/api"
+	spineapi "github.com/enbility/spine-go/api"
+	"github.com/enbility/spine-go/model"
+)
+
+type CemOHPCFInterface interface {
+	api.UseCaseInterface
+
+	SmartEnergyManagementData(entity spineapi.EntityRemoteInterface) (
+		smartEnergyManagementData model.SmartEnergyManagementPsDataType, resultErr error)
```

Comment:
Right now the API tries to avoid returning SPINE data models to the application, because these models are way too complex especially in this case. Instead it would be great to provide a simplified data structure which contains only the data required by the application.

Would it be possible to for you to design such a data structure here?

Reply:
Agreed! Im not 100% sure how this structure should look like, and one of the issues is I cant fully use OHPCF until other use cases are completed (we need to be able to set temperature and configure the mode at least).

Once these other usecases are completed and I have a better idea of what the top level API should look like i'll update this.